packages/jaeger-ui/src/components/SearchTracePage/index.jsx

```diff
@@ -246,9 +253,10 @@ const stateServicesXformer = memoizeOne(stateServices => {
 
 // export to test
 export function mapStateToProps(state) {
-  const { embedded, router, services: stServices, traceDiff } = state;
+  const { embedded, router, services: stServices, traceDiff, config } = state;
   const query = getUrlState(router.location.search);
   const isHomepage = !Object.keys(query).length;
+  const { forbidNewPage, disableFileUploadControl } = config;
```

Comment:
what guarantees that `config` won't be undefined here?

Reply:
As far as I was able to see, the default configuration is always being loaded (as long as the file is available under the path "constants/default-config"). So it should be defined. Would it be better when I take care of the case that config is being undefined? Like with an if and if it is undefined providing default values?